lib/normalizer.c

```diff
@@ -3428,6 +3428,23 @@ grn_nfkc_normalize_unify_romaji(grn_ctx *ctx,
                                     n_unified_characters);
 }
 
+static const unsigned char *
+grn_nfkc_normalize_unify_iteration_mark(grn_ctx *ctx,
+                                        const unsigned char *start,
+                                        const unsigned char *current,
+                                        const unsigned char *end,
+                                        size_t *n_used_bytes,
+                                        size_t *n_used_characters,
+                                        unsigned char *unified_buffer,
+                                        size_t *n_unified_bytes,
+                                        size_t *n_unified_characters,
+                                        void *user_data)
+{
+  /* TODO: Implement iteration mark (踊り字) expansion.
+   * For now, just return the current position unchanged. */
```

Comment:
How about implementing the easiest one pattern instead of nothing?
It seems that we can't test whether this works or not without implementation.